dask/array/_array_expr/_collection.py

```diff
@@ -655,3 +656,43 @@ def rechunk(
     return new_collection(
         x.expr.rechunk(chunks, threshold, block_size_limit, balance, method)
     )
+
+
+def from_array(
+    x,
+    chunks="auto",
+    lock=False,
+    asarray=None,
+    fancy=True,
+    getitem=None,
+    meta=None,
+    inline_array=False,
+):
```

Comment:
Do we have docs for this somewhere we can copy?

Reply:
good point, added